### DISCUSSION TITLE
Room 9 - Series 4

### DISCUSSION THREAD

**Discussion by awenasangster** [IBAHCM/RPiR#278] [Archive]
@awenasangster Awena Sangster: https://github.com/IBAHCM/AwenaSangsterBCI
@mikatakahashi Mika Takahashi: https://github.com/IBAHCM/MikaTakahashiBCI
@McGhee-Will Will McGhee: https://github.com/IBAHCM/BCIdata

**Comment by awenasangster** [comment]
Got you both!

**Comment by mikatakahashi** [comment]
my repo link - https://github.com/IBAHCM/MikaTakahashiBCI

**Comment by McGhee-Will** [comment]
Cheers Awena!
